Utilities/CodeGeneration/EdFi.Ods.CodeGen/Providers/Impl/DomainModelDefinitionProvidersProvider.cs

```diff
@@ -62,28 +65,44 @@ public IEnumerable<IDomainModelDefinitionsProvider> DomainModelDefinitionProvide
             var domainModelDefinitionsByPath =
                 new Dictionary<string, IDomainModelDefinitionsProvider>(StringComparer.InvariantCultureIgnoreCase);
 
-            string edfiOdsImplementationApplicationPath = _solutionPath;
+            string edFiOdsImplementationApplicationPath = _solutionPath;
 
             string edFiOdsApplicationPath = _solutionPath.Replace(
                 CodeRepositoryConventions.EdFiOdsImplementationFolderName,
                 CodeRepositoryConventions.EdFiOdsFolderName);
 
-            if (_includePluginsProvider.IncludePlugins() && Directory.Exists(_extensionsPath))
+            directoriesToEvaluate = GetProjectDirectoriesToEvaluate(edFiOdsImplementationApplicationPath)
+                .Concat(GetProjectDirectoriesToEvaluate(edFiOdsApplicationPath))
+                .ToArray();
+
+            var extensionPaths = _extensionLocationPluginsProviderProvider.GetExtensionLocationPlugins();
+
+            foreach (var extensionPath in extensionPaths)
```

Comment:
Can this be converted to a linq statement?

Reply:
Also we should have a test that validates this logic please

Reply:
Added unit test method